tests/test_internals/test_movie_interface.py

```diff
@@ -218,3 +219,54 @@ def test_movie_stub(self):
             for no in range(len(metadata["Behavior"]["Movies"])):
                 movie_interface_name = metadata["Behavior"]["Movies"][no]["name"]
                 assert mod[movie_interface_name].data.shape[0] == 10
+
+    def test_movie_irregular_timestamps(self):
+        timestamps = [1, 2, 4]
+        conversion_opts = dict(
+            Movie=dict(starting_times=self.starting_times, timestamps=timestamps, external_mode=True)
+        )
+
+        self.nwb_converter.run_conversion(
+            nwbfile_path=self.nwbfile_path,
+            overwrite=True,
+            conversion_options=conversion_opts,
+            metadata=self.metadata,
+        )
+
+        with NWBHDF5IO(path=self.nwbfile_path, mode="r") as io:
+            nwbfile = io.read()
+            acquisition_module = nwbfile.acquisition
+            metadata = self.nwb_converter.get_metadata()
+            for movie_metadata in metadata["Behavior"]["Movies"]:
+                movie_interface_name = movie_metadata["name"]
+                np.array_equal(timestamps, acquisition_module[movie_interface_name].timestamps[:])
```

Comment:
Does this work?
Shouldn't it be?
https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html